en/docs/learn/multi-channel-self-registration-and-account-confirmation.md

```diff
@@ -266,6 +296,9 @@ For more details see,
 
 WSO2 Identity Server provides the functionality to confirm the user account internally or externally.
 
+!!! note
+    To try out self-registration, above steps needs to be completed. 
```

Comment:
!!! tip "Before you begin"
    Complete the steps given above before trying out self-registration.

Reply:
Fixed: d72b99b72fc947f18610188db171158cd99f8f08